crates/telio-nurse/src/rtt/ping.rs

```diff
@@ -59,20 +60,15 @@ impl Ping {
         target: (PublicKey, DualTarget),
         results_tx: mpsc::Sender<(PublicKey, DualPingResults)>,
     ) {
-        // TODO this needs some refinement
         let dpr = self.perform_average_rtt(&target.1).await;
 
-        telio_log_debug!(
-            "Ping results: {:?}, no_of_tries: {:?}",
-            dpr,
-            self.no_of_tries
-        );
+        telio_log_debug!("{:?}, no_of_tries: {}", dpr, self.no_of_tries);
 
         let _ = results_tx.send((target.0, dpr)).await;
     }
 
     async fn perform_average_rtt(&self, target: &DualTarget) -> DualPingResults {
-        let mut dpresults = DualPingResults { v4: None, v6: None };
+        let mut dpresults = DualPingResults::default();
 
         match target.get_targets() {
             Ok(t) => {
```

Comment:
I think if you remove the refinement comment maybe it would be nice to refactor it a bit more by occasion? E.g. I guess it would be nice to decompose the tuple in the match branch instead of using `t.0` and `t.1`

Reply:
Not relevant to this PR, it's just a backport